### ISSUE TITLE
[BUG]: fail to start examor-server-1 container

### ISSUE THREAD

**Issue by JokinYang** [codeacme17/examor#21]
### Description of the bug

Unable up the compose env, because of fail to start the examor-server-1 .
output log as below:
```
2023-09-01 14:27:57 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:27:58 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:27:59 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:00 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:02 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:05 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:12 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:25 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:28:51 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:29:43 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:30:43 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:31:43 /usr/bin/env: ‘bash\r’: No such file or directory
2023-09-01 14:32:44 /usr/bin/env: ‘bash\r’: No such file or directory
```
the command I used :
```shell
git clone https://github.com/codeacme17/examor.git
cd examor
docker-compose up

```
version I used：
```shell
> git log -n 1
commit 6fec558a4667a002cd18de3f563ab29e7523169e (HEAD -> main, origin/main, origin/HEAD)
Author: codeacme17 <vir_kings@qq.com>
Date:   Thu Aug 31 21:08:51 2023 +0800

    fix: fix some bugs
```

**Comment by codeacme17** [comment]
Ok, thanks for the issue, I'll check why this bug happens

**Comment by codeacme17** [comment]
Try replace python:3.11.3 to python:3.11.3-alpine in /server/DockerFile. Then rerun the docker-compose. Maybe will fix this issue.

**Comment by JokinYang** [comment]
> @JokinYang, Try replace `python:3.11.3` to `python:3.11.3-alpine` in /server/DockerFile. Then rerun the docker-compose. Maybe will fix this issue.

I tried using `python:3.11.3-alpine` as the base image. error `/usr/bin/env: ‘bash\r’: No such file or directory` not occured, but it fails at installing numpy due to `[Errno 2] No such file or directory: 'gcc'`.   

 ----

Fixed this problem by removing the `\r` character behind the `#!/usr/bin/env bash` in the server/wait-for-it.sh

Thanks for your atteneion!
